src/styles/stars.scss

```diff
@@ -0,0 +1,31 @@
+.card__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+
+  &-reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
```

Comment:
It shouldn't be here - part of the card BEM block. BTW, card__container is not the best name here. It might be smth like card__reviews-container